src/Handler/Locator/InMemoryLocator.php

```diff
@@ -15,7 +15,7 @@
  *      $inMemoryLocator->addHandler($myHandler, 'My\TaskAddedCommand');
  *
  *      // Returns $myHandler
- *      $inMemoryLocator->getHandlerForCommand(new My\TaskAddedCommand());
+ *      $inMemoryLocator->getHandlerForCommand('My\TaskAddedCommand');
```

Comment:
That is an allowed notation; I have explicitly used the pre-php 5.5 notation because this library its minimum version is not 5.5